protocol/0056-REWA-rewards_overview.md

```diff
@@ -42,6 +41,32 @@ For reward metrics relating to trading, an individual must meet the [staking req
 
 For reward transfers where the [scope](./0057-TRAN-transfers.md#recurring-transfers-to-reward-accounts) is set to teams, each party must meet the minimum time in team requirement. That is, given a party has been in a team for $N$ epochs, if $N$ is strictly less than the network parameter `rewards.minimumEpochsInTeam` (an integer defaulting to `0`) their reward metric is set to `0`.
 
+## Publishing Interim Reward Information
+
+Although rewards are distributed at the end of an epoch, to give users of the protocol an understanding of what rewards they are likely to receive at the next distribution event, the protocol must publish interim data during the epoch to populate data nodes.
+
+For each game, each entities (individual or team) **current reward metric** and rank in the game is to be published at the specified frequency. Note, the actual reward metric should be published rather than the underlying data in the current epoch, this is important for games where the window length is greater than `1`.
```

Comment:
@cdummett I am not seeing where we specify the frequency, probably should be a network parameter so we can fine-tune the load this creates on the data node.

Reply:
The frequencies are in the table below as per what we decided in the research hour.

We discussed adding a network parameter for any reward metrics which were only updated at the end of an epoch but having double checked these (`Liquidity Fees`, `Market Creation`, `Validator Ranking`) the underlying events for these rewards only happen at the end of the epoch anyway so we can't calculate them at a faster frequency.